available_images.md

```diff
@@ -142,12 +142,23 @@ SageMaker Framework Graviton Containers (SM support only)
 
 NVIDIA Triton Inference Containers (SM support only)
 ============================
-**Note**:
+**Versions 23.12 and onwards**:
+1. Starting version 23.12, onwards, the sagemaker tritonserver is available in a different set of accounts than the previous ones. The new accounts are now the same as other DLCs listed on this page, making it easier to switch containers going forward. These accounts are listed in section '# Available Deep Learning Containers Images' above.
+2. Alternately, they can be obtained programmatically from the sagemaker python sdk as:
+```
+    uri = image_uris.retrieve(
+        framework=triton_framework, region=region, version=version, instance_type=instance_type
+    )
+```
+where, for example `triton_framework=sagemaker-tritonserver`, and `region=us-west-2`, `version=23.12`, `instance_type=ml.g5.12xlarge`.
+3. Unavailable versions: `24.02`.
+
+**Versions prior to 23.12**:
 
 1. The following versions of the 23.`<XY>` container are supported: `23.01, 23.02, 23.03, 23.05, 23.06, 23.07, 23.08, 23.09, 23.10 and 23.12`.
```

Comment:
If we are already writing out the versions in support, would it be more helpful to just use the same table structure that we use for other DLCs? Otherwise it might create a bunch of extra work for a customer to construct the image URI of choice